DOCUMENTATION.md

```diff
@@ -117,21 +152,31 @@ This document provides detailed information about each service module and its re
       * [Get jobs](#get-jobs)
       * [Create Direct Request job](#create-direct-request-job)
     * [Service alias: `linkToken`](#service-alias-linktoken)
-    * [Methods](#methods-10)
+    * [Methods](#methods-12)
       * [Deploy contract](#deploy-contract)
       * [Transfer](#transfer)
       * [Get allowance](#get-allowance)
       * [Increase approval](#increase-approval)
       * [Decrease approval](#decrease-approval)
     * [Service alias: `operator`](#service-alias-operator)
-    * [Methods](#methods-11)
+    * [Methods](#methods-13)
       * [Deploy contract](#deploy-contract-1)
       * [Set authorized sender](#set-authorized-sender)
     * [Service alias: `drConsumer`](#service-alias-drconsumer)
-    * [Methods](#methods-12)
+    * [Methods](#methods-14)
       * [Deploy contract](#deploy-contract-2)
       * [Request data](#request-data)
       * [Get latest answer](#get-latest-answer)
+    * [Service alias: `functionsConsumer`](#service-alias-functionsconsumer)
```

Comment:
The `FunctionsConsumer.sol` contracts are examples only.  The only hard requirements are that it inherits from `FunctionsClient` and that it implements `fulfillRequest(bytes32 requestId, bytes memory response, bytes memory err) ` 

Rather than include the Functions Consumer (which will make people feel that they HAVE to use it) it is probably better to have the user "BYO" consumer contract, and pass a path to it?

Reply:
That contract is in the **sandbox** section and has a disclaimer that avoids user's to use it in a production environment (https://github.com/koteld/hardhat-chainlink/blob/5155fb642a682a16ca955cb48ba0081d5bbca6bf/contracts/FunctionsConsumer.sol#L8), therefore it is intended more for testing rather than for use in a user project's SC stack.
If we want to make wrappers for custom Functions Consumer contracts and make it available to call methods like "sendRequest" we have to introduce another Interface on top of FunctionsClient contract, otherwise it will be useless.